src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -98,16 +100,19 @@
   private PlatformService platformService;
   private String baseClanWebsite;
   private ExecutorService executorService = Executors.newFixedThreadPool(2);
+  private FxmlLoader fxmlLoader;
+
 
   @Inject
   // TODO reduce dependencies, rely on eventBus instead
   public ChatUserItemController(PreferencesService preferencesService, AvatarService avatarService,
                                 CountryFlagService countryFlagService, ChatService chatService,
                                 ReplayService replayService, I18n i18n, UiService uiService,
-                                ReportingService reportingService,
+                                ReportingService reportingService, FxmlLoader fxmlLoader,
                                 JoinGameHelper joinGameHelper, EventBus eventBus, ClanService clanService,
                                 PlayerService playerService, PlatformService platformService,
                                 @Value("${clan.clanWebpagesBaseUrl}") String baseClanWebsite) {
+    this.fxmlLoader = fxmlLoader;
```

Comment:
My fault; `FxmlLoader` shouldn't be used anymore, use `UiService` instead. I added `@Deprecated` now :-)